lib/CMakeLists.txt

```diff
@@ -4,8 +4,9 @@ endif ()
 
 add_library(preopen SHARED
 	libpreopen.c
-	interceptAccess.c
```

Comment:
Why have these files been renamed? In fact, they are each so small that we could probably combine them into a single `intercept.c`, or even better, `wrappers.c`.